src/clr-angular/layout/tabs/tab-content.ts

```diff
@@ -12,7 +12,14 @@ let nbTabContentComponents: number = 0;
 @Component({
   selector: 'clr-tab-content',
   template: `
+    <ng-template #tabContentProjectedRef>
```

Comment:
Restored from git log. It corresponds to a "template" var below, that was previously empty.